src/components/NotificationBar/NotificationBar.js

```diff
@@ -45,7 +45,7 @@ class NotificationBar extends Component {
         <div className="notification-bar--info">
           <div className="notification-bar--alert">{this.props.mainText}</div>
           <div className="notification-bar--timer">
-            {this.props.autoActionWord + ' in' + ' ' + this.state.countdown}s
+            {this.props.autoActionWord + ' in ' + this.state.countdown}s
```

Comment:
its in a div so you can just type I actually meant its best to do
`{this.props.autoActionWord} in {this.state.countdown}s`